tests/test_hyperbolic.py

```diff
@@ -469,8 +470,8 @@ def test_scaled_squared_norm(self):
     def test_scaled_distance(self):
         point_a_intrinsic = gs.array([1, 2, 3])
         point_b_intrinsic = gs.array([4, 5, 6])
-        point_a = self.space.intrinsic_to_extrinsic_coords(point_a_intrinsic)
-        point_b = self.space.intrinsic_to_extrinsic_coords(point_b_intrinsic)
+        point_a = self.space.from_coordinates(point_a_intrinsic, "intrinsic")
+        point_b = self.space.from_coordinates(point_b_intrinsic, "intrinsic")
         scale = 2
         default_space = Hyperbolic(dimension=self.dimension)
```

Comment:
I know this is not your line, but here self.space could be used, instead of re-instantiating a new Hyperbolic object.

Reply:
yes, I change it